tools/@aws-cdk/ubergen/lib/index.ts

```diff
@@ -7,20 +7,21 @@ import * as awsCdkMigration from 'aws-cdk-migration';
 import * as fs from 'fs-extra';
 
 // The directory where our 'package.json' lives
-const MONOPACKAGE_ROOT = process.cwd();
 
-const ROOT_PATH = findWorkspacePath();
-const UBER_PACKAGE_JSON_PATH = path.join(MONOPACKAGE_ROOT, 'package.json');
-
-const EXCLUDED_PACKAGES = ['@aws-cdk/example-construct-library'];
+export interface Config {
+  monoPackageRoot: string;
+  rootPath: string;
+  uberPackageJsonPath: string;
+  excludedPackages: string[];
+}
 
-export async function main() {
-  console.log(`🌴  workspace root path is: ${ROOT_PATH}`);
-  const uberPackageJson = await fs.readJson(UBER_PACKAGE_JSON_PATH) as PackageJson;
-  const libraries = await findLibrariesToPackage(uberPackageJson);
-  await verifyDependencies(uberPackageJson, libraries);
-  await prepareSourceFiles(libraries, uberPackageJson);
-  await combineRosettaFixtures(libraries, uberPackageJson);
+export async function main(config: Config) {
```

Comment:
Do we need these changes to main? I don't see them used in the remodel cli.ts, but maybe I am missing something.